src/vma/config_parser.c

```diff
@@ -70,7 +70,7 @@
 
 /* Copy the first part of user declarations.  */
 /* Line 371 of yacc.c  */
-#line 39 "config_parser.y"
+//#line 39 "config_parser.y"
```

Comment:
For all this file better using 
```
/* XXX */
```
rather than
```
// XXX
```
to be consistent with the comments in the original file. Also // is rather a C++ comment and this is a C file